src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -189,22 +190,21 @@ public abstract class GraphQLHttpMiddleware
     public GraphQLHttpMiddleware(
         RequestDelegate next,
         IGraphQLTextSerializer serializer,
-        GraphQLHttpMiddlewareOptions options /*,
-        IEnumerable<IWebSocketHandler>? webSocketHandlers = null */)
+        GraphQLHttpMiddlewareOptions options,
+        IEnumerable<IWebSocketHandler>? webSocketHandlers = null)
     {
         _next = next ?? throw new ArgumentNullException(nameof(next));
         _serializer = serializer ?? throw new ArgumentNullException(nameof(serializer));
         Options = options ?? throw new ArgumentNullException(nameof(options));
-        //_webSocketHandlers = webSocketHandlers;
-        //_userContextBuilderForWebSockets = new UserContextBuilder<IDictionary<string, object?>>(BuildUserContextAsync);
+        _webSocketHandlers = webSocketHandlers;
+        _userContextBuilderForWebSockets = new UserContextBuilder<IDictionary<string, object?>>(BuildUserContextAsync);
```

Comment:
🤔 It looks a bit weird to wrap existing method into `UserContextBuilder` object when inside `BuildUserContextAsync` you pull `IUserContextBuilder` from DI. Why not use `context.RequestServices.GetService<IUserContextBuilder>()` for `await selectedHandler.ExecuteAsync` line?

Reply:
Because I want it to call the protected method within this class.